presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.tests.hive;
+
+import com.teradata.tempto.ProductTest;
+import com.teradata.tempto.query.QueryExecutor;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.tests.TestGroups.HIVE_CONNECTOR;
+import static com.facebook.presto.tests.TestGroups.QUARANTINE;
+import static com.facebook.presto.tests.utils.QueryExecutors.connectToPresto;
+import static com.teradata.tempto.assertions.QueryAssert.assertThat;
+import static java.lang.String.format;
+
+public class TestGrantRevoke
+    extends ProductTest
+{
+    /* This test has been quarantined since hive.security property is NOT set by default and
+     * doing so may affect other tests.
+     *
+     * Pre-requisites for the test:
+     * (1) In hive.properties file, set this property: hive.security=sql-standard;
+     * (2) tempto-configuration.yaml file should have two connections to Presto server:
+     * "prestoConfigUserAlice" that has "jdbc_user: alice" and
```

Comment:
does user have to prepare above configuration manually? If so then I don't think that FB guys would like it.